frontend/language/src/nb.json

```
@@ -34,10 +34,12 @@
   "app_create_release_errors.check_status_on_build_error": "Oi! Vi opplever en teknisk feil og får derfor ikke oppdatert status på bygget. Vi forsøker å sjekke status på nytt...",
   "app_create_release_errors.fetch_release_failed": "Huff da, vi opplever en teknisk feil som ikke gjør det mulig for deg å bygge versjoner eller deploye en applikasjon til miljøene akkurat nå. Prøv igjen senere. Dersom problemet vedvarer, kontakt <0 href=\"mailto:tjenesteeier@altinn.no\">Altinn servicedesk</0>.",
   "app_create_release_errors.technical_error_code": "Teknisk feilkode",
+  "app_data_modelling.fields_information": "Vennligst merk at kun objekter kan tilordnes felter i datamodelleringsverktøyet. Tekst, heltall, desimaltall og ja/nei. støtter ikke egne felter. For å løse dette, velg enten en annen node som er et objekt eller konverter gjeldende node til et objekt.",
```

Kunne vi unngått å bruke ordet "vennligst" her? Jeg kan ikke se at det gir noen verdi. Foreslår også at vi bruker ordet "element" i stedet for "node" for å redusere bruken av intern sjargong.
```suggestion
  "app_data_modelling.fields_information": "Kun objekter kan tilordnes felter i datamodelleringsverktøyet. Tekst, heltall, desimaltall og ja/nei støtter ikke egne felter. For å løse dette, velg et element som er et objekt eller konverter gjeldende element til et objekt.",
```